sdlib/d/gc/region.d

```diff
@@ -63,6 +65,11 @@ private:
 	size_t minAddress = AddressSpace;
 	size_t maxAddress = 0;
 
+	// Blocks that have been allocated, used for tracking general memory usage.
+	size_t nBlocks;
```

Comment:
Maybe the initial diff should just track this, and add test for it.

Simple, easy, obviously needed, mergeable right away.

Reply:
ok